app/models/alchemy/essence_credit.rb

```diff
@@ -12,4 +16,52 @@ def license
     return db_license if inverted_licenses.key?(db_license)
     'CNE' # unable to determine
   end
+
+  # Europeana record for +europeana_record_id+
+  #
+  # @return [Europeana::Record]
+  def europeana_record
+    return @europeana_record if instance_variable_defined?(:@europeana_record)
+    @europeana_record = Europeana::Record.new(europeana_record_id)
+  end
+
+  def europeana_record_id
+    return @europeana_record_id if instance_variable_defined?(:@europeana_record_id)
+    @europeana_record_id = Europeana::Record.id_from_portal_url(url)
+  end
+
+  def has_annotation_target?
+    europeana_record_id.present?
+  end
+
+  def annotation
+    @annotation ||= Europeana::Annotation.new(annotation_attributes)
+  end
+
+  def annotation_target
```

Comment:
These are not annotations of specific web resources, but annotations at the record level. Annotation target should just be the data.europeana.eu URI as a string, e.g. `"target": "http://data.europeana.eu/item/92062/BibliographicResource_1000126189360"`